test/calendar.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { test, Page } from "bigtest";
+import { Calendar as Component } from "@material-ui/pickers";
+import { Calendar, getCalendar } from "../src";
+import { getPickerRenderer } from "./helpers";
+import DateFnsUtils from "@date-io/date-fns";
+
+const renderComponent = getPickerRenderer(Component);
```

Comment:
Let's call this `renderCalendar` since it's more specific.

Reply:
Do we need to rename back the `Calendar as Component` in an import statement?